README.md

```diff
@@ -68,6 +68,23 @@ Due to the structure of this library, the css of the components is **not include
 
 Uses node version > **14.x.x** < **16.x.x**
 
+### To run locally
+
+```shell
+# Open storybook dev server on `http://localhost:6006/`
+yarn storybook
+```
```

Comment:
Perhaps we should run `yarn storybook` and **then** open storybook dev server? If the user opens the dev server **before** running the command, there will be nothing to see. Wouldn't that be a bad user experience?